examples/3rd-party-oauth-login/src/app.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-undef */
+// check eslint global config
+import "./assets/style.css";
+
+var loginBtn = document.getElementById("loginButton");
+var logoutBtn = document.getElementById("logoutButton");
+var loginText = document.getElementById("loginText");
+
+// Attach click event listeners
+loginBtn.addEventListener("click", startOAuthFlow);
+logoutBtn.addEventListener("click", handleLogout);
+
+// Add an event listener to the DOMContentLoaded event
+document.addEventListener("DOMContentLoaded", function () {
+  // Call the initialize function when the DOM content is loaded
+  initialize();
+});
+
+// Define a function containing the code you want to run each time index.html is opened
+async function initialize() {
+  logoutBtn.style.display = "none";
+  await isLoggedIn();
+}
+
+// This function displays the user login status on the UI
+async function displayLoginStatus(loggedIn) {
+  try {
+    var statusParagraph = document.getElementById("loginStatus");
+
+    if (loggedIn) {
+      statusParagraph.textContent =
+        "✅ User is logged in. " +
+        "🔄 If you want to run this flow again, use the logout button.";
+
+      // Hide the login button and show the logout button
+      loginBtn.style.display = "none";
+      loginText.style.display = "none";
+      logoutBtn.style.display = "block";
+    } else {
+      statusParagraph.textContent = "❌ User is not logged in ❌";
+      loginBtn.style.display = "block";
+    }
+    return;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+// This function checks the local storage for the user's id. Local storage is just for demo purposes.
+// It is recommended to implement your own storage for a production application.
+async function isLoggedIn() {
+  try {
+    // Check if the user is logged in by checking the local storage on the browser
+    const loggedIn = localStorage.isLoggedIn;
```

Comment:
does this work? Should it be `localStorage.getItem('isLoggedIn')`?

Reply:
changed it to

    const loggedIn = localStorage.getItem("isLoggedIn") === "true";

and this seems to work fine.